Source/Blazorise/Components/Offcanvas/Offcanvas.razor.cs

```diff
@@ -0,0 +1,484 @@
+    /// <returns>True if offcanvas can be closed.</returns>
+    private async Task<bool> IsSafeToClose()
+    {
+        var safeToClose = true;
+
+        if ( Closing != null )
+        {
+            var eventArgs = new OffcanvasClosingEventArgs( false, closeReason );
+
+            await Closing.Invoke( eventArgs );
+
+            if ( eventArgs.Cancel )
+            {
+                safeToClose = false;
+            }
+        }
+
+        return safeToClose;
+    }
+
+    /// <summary>
+    /// Handles the internal visibility states.
+    /// </summary>
+    /// <param name="visible">Visible state.</param>
+    private async Task SetVisibleState( bool visible )
+    {
+        state = state with { Visible = visible };
+
+        await HandleVisibilityStyles( visible );
+        await RaiseEvents( visible );
+    }
+
+    /// <inheritdoc/>
+    public Task BeginAnimation( bool visible )
+    {
+        if ( visible )
+        {
+            showing = true;
+
+            BackdropVisible = ShowBackdrop;
+        }
+        else
+        {
+            hiding = true;
+        }
+
+        DirtyClasses();
+        DirtyStyles();
+
+        return InvokeAsync( StateHasChanged );
+    }
+
+    /// <inheritdoc/>
+    public Task EndAnimation( bool visible )
+    {
+        if ( visible )
+        {
+            showing = false;
+        }
+        else
+        {
+            hiding = false;
+        }
+
+        DirtyClasses();
+        DirtyStyles();
+
+        BackdropVisible = ShowBackdrop && visible;
+
+        return InvokeAsync( StateHasChanged );
+    }
+
+    #endregion
+
+    #region Properties
+
+    /// <summary>
+    /// True if Offcanvas contains the <see cref="OffcanvasHeader"/> component.
+    /// </summary>
+    protected bool HasOffcanvasHeader { get; set; }
+
+    /// <summary>
+    /// True if Offcanvas contains the <see cref="OffcanvasBody"/> component.
+    /// </summary>
+    protected bool HasOffcanvasBody { get; set; }
+
+    /// <summary>
+    /// Gets a value indicating whether the Offcanvas is visible.
+    /// </summary>
+    protected internal bool IsVisible => state.Visible == true;
+
+    /// <summary>
+    /// Gets the reference to state object for this Offcanvas.
+    /// </summary>
+    protected internal OffcanvasState State => state;
+
+    /// <summary>
+    /// Returns true if the offcanvas backdrop should be visible.
+    /// </summary>
+    protected internal bool BackdropVisible = false;
+
+    /// <summary>
+    /// The injected JavaScript module for closable operations.
+    /// </summary>
+    [Inject] public IJSClosableModule JSClosableModule { get; set; }
+
+    /// <summary>
+    /// Gets or sets the visibility state of the Offcanvas.
+    /// </summary>
+    [Parameter] public bool Visible { get; set; }
+
+    /// <summary>
+    /// Event callback for when the visibility state of the Offcanvas changes.
+    /// </summary>
+    [Parameter] public EventCallback<bool> VisibleChanged { get; set; }
```

Comment:
Here, I'm seeing **Visible** parameters, but on the docs I see **Show** parameters.
![image](https://github.com/Megabit/Blazorise/assets/22283161/fcf66a27-fa6e-4669-8bcf-dacc4e7a6f00)